yarn.lock

```diff
@@ -465,13 +465,13 @@
     "@babel/helper-plugin-utils" "^7.8.3"
     "@babel/plugin-syntax-optional-chaining" "^7.8.0"
 
-"@babel/plugin-proposal-throw-expressions@^7.10.1":
-  version "7.10.1"
-  resolved "https://registry.yarnpkg.com/@babel/plugin-proposal-throw-expressions/-/plugin-proposal-throw-expressions-7.10.1.tgz#89bde971b6081bfecff832c811ad76c2337c6970"
-  integrity sha512-wyhTn1ebUbbphDJVVUpg1wikUOIW8Lg93ng0aDGv07+MxINPS4d0ju68JVH06W4oMnZTwgELA5AImrah9RULYA==
+"@babel/plugin-proposal-throw-expressions@^7.8.3":
```

Comment:
You might want to pull the latest from master and push it here so you are not changing the `yarn.lock` and reversing the upgrades previously made in master.

Reply:
@jhonnyoddball I pulled the latest and merged it in so I think this is fixed?